index.js

```diff
@@ -23,7 +23,7 @@ function create (opts) {
   var misc = subleveldown(db, 'misc', {valueEncoding: 'binary'})
   var keys = subleveldown(db, 'added-keys', {valueEncoding: 'binary'})
   var noContent = subleveldown(db, 'no-content', {valueEncoding: 'binary'})
-  var core = hypercore(db)
+  var core = hypercore(storage)
```

Comment:
for this use cause yes. var stream = protocol() ... feed.replicate({stream: stream})